TLPhotoPicker/Classes/TLAssetsCollection.swift

```diff
@@ -201,10 +203,6 @@ public struct TLPHAsset {
             }
             return PHImageManager.default().requestImageData(for: phAsset, options: requestOptions, resultHandler: { (data, uti, orientation, info) in
                 do {
-                    var data = data
-                    if convertLivePhotosToJPG == true, let imgData = data, let rawImage = UIImage(data: imgData)?.upOrientationImage() {
```

Comment:
Even if this is not live photo, and convertLivePhotosToJPG is true, this tries to create jpeg data which is inefficient.